orderapi/src/main/java/fi/hel/verkkokauppa/order/api/data/subscription/SubscriptionDto.java

```diff
@@ -51,7 +51,7 @@
     private String priceVat;
     private String priceGross;
     // Relations data
-    private Set<String> relatedOrderIds; // "Main" subscription contains all orders created from it.
+    private String orderId;
```

Comment:
OrderId should be deleted from subscription, it is not needed since order now has subscriptionId

Reply:
When subscription is created from order the order doesn't yet have subscription id. This is the "master" order id for subscription from where subscription was created. At least this is how i assume..

Reply:
That would make sense, then maybe renaming the variable and/or adding comment to better make this clear would be good